### DISCUSSION TITLE
getting error during event creation

### DISCUSSION THREAD

**Discussion by sunilcpr** [hackjunction/hackplatform#455] [General]
Please anyone help me to figure out ,when I create an event i getting error 

Unable to save changes

theme: undefined
undefined: this.$__schema.paths[p].splitPath is not a function
travelGrantConfig: undefined
registrationConfig: undefined

**Comment by Jussii** [comment]
Hey, we upgraded our mongoose version to 5.12. that broke the build. We have now downgraded it back to 5.9. #457 and everything should be working as before.

In the future if you want to report a bug. Please use https://github.com/hackjunction/JunctionApp/issues/new/choose bug report issue form. Thank you
